### PR TITLE
Correct examples to use -featureID

### PR DESCRIPTION
**Pull request by timwarner-msft**
In my use of this command, passing the -Feature parameters results in an error. Using -featureID work as expected. I also cleaned up the PowerShell code to add missing parameter names and get rid of the "PS C:\> prefix, which to me is wholly unnecessary. 

-------
cc: @JasonGerend

### CODE REVIEW

**Comment by timwarner-msft** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2012r2-ps/servermigration/Import-SmigServerSetting.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ For online Help about the Windows Server Migration Tools cmdlets, see http://go.
 
 ### EXAMPLE 1
 ```
-PS C:\> Import-SmigServerSetting -Feature "DHCP" -User All -Group -Path "c:\temp\store" -Verbose
+Import-SmigServerSetting -featureID "DHCP" -User All -Group -Path "c:\temp\store" -Verbose
```
Comment (sdwheeler) [inline docset/winserver2012r2-ps/servermigration/Import-SmigServerSetting.md:38]: Use the proper Pascal Case name of the parameter.
```suggestion
Import-SmigServerSetting -FeatureID 'DHCP' -User All -Group -Path 'c:\temp\store' -Verbose
```

File: docset/winserver2012r2-ps/servermigration/Import-SmigServerSetting.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ By using the -Verbose parameter, the command also displays detailed information
 
 ### EXAMPLE 2
 ```
-PS C:\> Import-SmigServerSetting -IPConfig All -SourcePhysicalAddress "00-13-D3-F7-A1-3A","00-13-D3-F7-A1-4A" -TargetPhysicalAddress "11-13-D3-F7-A1-3A","11-13-D3-F7-A1-4A" -Path "c:\temp\store" -Password (Read-Host "Enter a Password:" -AsSecureString)-Verbose
+Import-SmigServerSetting -IPConfig All -SourcePhysicalAddress "00-13-D3-F7-A1-3A","00-13-D3-F7-A1-4A" -TargetPhysicalAddress "11-13-D3-F7-A1-3A","11-13-D3-F7-A1-4A" -Path "c:\temp\store" -Password (Read-Host -Prompt "Enter a Password:" -AsSecureString)-Verbose
```
Comment (sdwheeler) [inline docset/winserver2012r2-ps/servermigration/Import-SmigServerSetting.md:53]: For readability this should be changed to use splatting.
```suggestion
$parameters = @{
    IPConfig = 'All'
    SourcePhysicalAddress = '00-13-D3-F7-A1-3A','00-13-D3-F7-A1-4A'
    TargetPhysicalAddress = '11-13-D3-F7-A1-3A','11-13-D3-F7-A1-4A'
    Path = 'c:\temp\store'
    Password (Read-Host -Prompt 'Enter a Password:' -AsSecureString)
    Verbose = $true
}
Import-SmigServerSetting @parameters
```

File: docset/winserver2012r2-ps/servermigration/Import-SmigServerSetting.md [hunk 3]
```diff
@@ -65,8 +65,8 @@ By using the -Verbose parameter, the command also displays detailed information
 
 ### EXAMPLE 3
 ```
-PS C:\> $c = Get-SmigServerFeature -Path "c:\temp\store"
-PS C:\> Import-SmigServerSetting -Feature $c -Path "c:\temp\store" -Verbose
+$c = Get-SmigServerFeature -Path "c:\temp\store"
+Import-SmigServerSetting -featureID $c -Path "c:\temp\store" -Verbose
```
Comment (sdwheeler) [inline docset/winserver2012r2-ps/servermigration/Import-SmigServerSetting.md:69]:
```suggestion
$c = Get-SmigServerFeature -Path 'c:\temp\store'
Import-SmigServerSetting -FeatureID $c -Path 'c:\temp\store' -Verbose
```

File: docset/winserver2012r2-ps/servermigration/Import-SmigServerSetting.md [hunk 4]
```diff
@@ -83,7 +83,7 @@ By using the -Verbose parameter, the command also displays detailed information
 
 ### EXAMPLE 4
 ```
-PS C:\> Get-SmigServerFeature -Path "c:\temp\store" | Import-SmigServerSetting -Path "c:\temp\store" -Verbose
+Get-SmigServerFeature -Path "c:\temp\store" | Import-SmigServerSetting -Path "c:\temp\store" -Verbose
```
Comment (sdwheeler) [inline docset/winserver2012r2-ps/servermigration/Import-SmigServerSetting.md:86]: Use single quotes
```suggestion
Get-SmigServerFeature -Path 'c:\temp\store' | Import-SmigServerSetting -Path 'c:\temp\store' -Verbose
```

File: docset/winserver2012r2-ps/servermigration/Import-SmigServerSetting.md [hunk 5]
```diff
@@ -100,8 +100,8 @@ By using the -Verbose parameter, the command also displays detailed information
 
 ### EXAMPLE 5
 ```
-PS C:\> $pass = ConvertTo-SecureString -String "password" -AsPlainText -Force
-PS C:\> Import-SmigServerSetting -User All -Password $pass -Path "c:\store" -Verbose
+$pass = ConvertTo-SecureString -String "password" -AsPlainText -Force
+Import-SmigServerSetting -User All -Password $pass -Path "c:\store" -Verbose
```
Comment (sdwheeler) [inline docset/winserver2012r2-ps/servermigration/Import-SmigServerSetting.md:104]:
```suggestion
$pass = ConvertTo-SecureString -String 'password' -AsPlainText -Force
Import-SmigServerSetting -User All -Password $pass -Path 'c:\store' -Verbose
```

**Review by sdwheeler — CHANGES_REQUESTED** [review]
@timwarner-msft See my suggestions. Also, the other versions of this probably need the same updates. Please update the other versions as well.

**Review by timwarner-msft — COMMENTED** [review]
Thanks for your insights, @sdwheeler!

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**